src/uproot/writing/writable.py

```diff
@@ -109,7 +109,7 @@ def recreate(file_path, **options):
         "initial_streamers_bytes", create.defaults["initial_streamers_bytes"]
     )
     uuid_function = options.pop("uuid_function", create.defaults["uuid_function"])
-    if len(options) != 0:
+    if options:
```

Comment:
This is one of the few things that I'm self-consciously against the grain with in Python: the truthiness rules that turn any Python object into a boolean when it's in the position of a predicate seem very dangerous to me. I would personally prefer to be explicit in all of these cases.

```suggestion
    if len(options) != 0:
```

But since the Pythonic thing is to rely on truthiness, this is negotiable.

Reply:
IMO, since the type is fixed (I checked for any of these I changed), then this reads better IMO ("if there are options" is how I'd read it in English), and is potentially faster too, since it's calling fewer functions.

Once you start statically typing, this truthiness rule is much less bothersome. If you know the type, it's easy to safely use truthiness.

Reply:
If you still want these, feel free to self-accept the suggestions, it's your project :)